pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -82,6 +82,9 @@ const (
 	PriorityIndex = "priority"
 	// ClusterGroupIndex is used to index ClusterNetworkPolicies by ClusterGroup names.
 	ClusterGroupIndex = "clustergroup"
+	// ServiceAccountsPeerIndex is used to index ClusterNetworkPolicies by ServiceAccount labels.
+	ServiceAccountsPeerIndex = "hasServiceAccountsPeer"
+	HasServiceAccountsPeer   = "true"
```

Comment:
I feel like the index name should be `HasServiceAccountsPeerIndex` and the `HasServiceAccountsPeer` constant is not really needed (you can just use the boolean value in the code).

Reply:
Changed.